.gitignore

```diff
@@ -180,7 +184,7 @@ _pkginfo.txt
 
 # Others
 ClientBin/
-~$*
+~*
```

Comment:
What's this?

Reply:
Again, update from upstream. This specifically is a POSIX file-system specific ignore.